graviti/manager/draft.py

```diff
@@ -250,11 +250,14 @@ def __init__(self, dataset: "Dataset") -> None:
 
     def _generate(
         self,
-        state: Optional[str],
+        state: str,
         branch: Optional[str],
```

Comment:
do we need to change the type of `branch` to `str`?

Reply:
but we need to set `branch = None` here